main/src/main/scala/sbt/Keys.scala

```diff
@@ -451,6 +451,7 @@ object Keys {
   val compatibilityWarningOptions = settingKey[CompatibilityWarningOptions]("Configures warnings around Maven incompatibility.").withRank(CSetting)
   val conflictWarning = settingKey[ConflictWarning]("Configures warnings for conflicts in dependency management.").withRank(CSetting)
   val conflictManager = settingKey[ConflictManager]("Selects the conflict manager to use for dependency management.").withRank(CSetting)
+  val conflictManagers = settingKey[Seq[ConflictManager]]("Selects the conflict managers to use for dependency management.").withRank(CSetting)
```

Comment:
If we introduce this, then we should add a deprecation warning on the old setting.

I know there hasn't been any scaladoc on most of the settings, but IMHO the explanation you put in this PR would be a perfect scaladoc here.